.github/workflows/cd.yml

```diff
@@ -0,0 +1,30 @@
+name: CD
+
+on:
```

Comment:
See how we do it on the JSON Schema. What you have here allows even releases from tags on branches other than `main`, which would be a security issue